go_pprof_scraper/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2b13f6b1-d3ba-4254-93c0-2ceaf783cd85",
+  "app_id": "go-pprof-scraper",
+  "display_on_public_website": false,
```

Comment:
Do you want this integration to be "private" for now?

Reply:
The only consequence of leaving this value as "false" would be that there's not a panel on the integrations page, right? Is it easy to change later? If so, I'm okay leaving it private for now. I've tested this integration myself, but it wouldn't hurt to have some other users internally give it a test run before we widely advertise it.

Reply:
Yup! let's keep it as `false` then because yes it's easy to change later